inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasUimaParser.java

```diff
@@ -243,8 +243,9 @@ private int indexFeatureValue(MtasTokenCollection aTokenCollection, AnnotationFe
             String annotationUiName)
     {
         int mtasId = aMtasId;
-        
-        String featureValue = WebAnnoCasUtil.getFeature(aAnnotation, aFeature.getName());
+        FeatureSupport<?> featSup = featureSupportRegistry.getFeatureSupport(aFeature);
```

Comment:
Mind that for concept features previously `WebAnnoCasUtil.getFeature(aAnnotation, aFeature.getName());` was indexing the URI at this point. If we switch to the UI value, then `indexConcept()` should probably be changed to index the URI, otherwise we do not index the URI anywhere meaning that URI-based search for concepts would no longer work. This would break the mention view on the KB page (which should be searching URI-based).